From c1dc146df69013040d5f22757fd48459886aab8e Mon Sep 17 00:00:00 2001
From: Alexander Martinz <amartinz@shiftphones.com>
Date: Mon, 9 Apr 2018 19:23:17 +0200
Subject: [PATCH 2/3] xml: device_profiles: correct XML tag

When specifying the package name, Android Studio gets really confused

Change-Id: I26e2b874c4a72f1f042eb581cb14f3f6e055d61c
Signed-off-by: Alexander Martinz <amartinz@shiftphones.com>
---
 res/xml/device_profiles.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/xml/device_profiles.xml b/res/xml/device_profiles.xml
index c5cec46e1..2b1bf5249 100644
--- a/res/xml/device_profiles.xml
+++ b/res/xml/device_profiles.xml
@@ -15,7 +15,7 @@
      limitations under the License.
 -->
 
-<profiles xmlns:launcher="http://schemas.android.com/apk/res-auto/com.android.launcher3" >
+<profiles xmlns:launcher="http://schemas.android.com/apk/res-auto">
 
     <profile
         launcher:name="Super Short Stubby"
@@ -193,4 +193,4 @@
         launcher:defaultLayoutId="@xml/default_workspace_5x6"
         />
 
-</profiles>
\ No newline at end of file
+</profiles>
-- 
2.15.1

